Fails to build from source: Creation method of abstract class cannot be public.

Can't build elementary-tweaks in Archlinux:
```
+ exec meson setup --prefix /usr --libexecdir lib --sbindir bin --buildtype plain --auto-features enabled --wrap-mode nodownload -D b_lto=true -D b_pie=true build
The Meson build system
Version: 0.52.0
Source dir: /build/switchboard-plug-elementary-tweaks-git/src/switchboard-plug-elementary-tweaks
Build dir: /build/switchboard-plug-elementary-tweaks-git/src/switchboard-plug-elementary-tweaks/build
Build type: native build
Project name: elementary-tweaks
Project version: 1.0
Appending CFLAGS from environment: '-march=native -O2 -pipe -fstack-protector-strong -fno-plt'
Appending LDFLAGS from environment: ['-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now']
Appending CPPFLAGS from environment: '-D_FORTIFY_SOURCE=2'
Appending CFLAGS from environment: '-march=native -O2 -pipe -fstack-protector-strong -fno-plt'
Appending LDFLAGS from environment: ['-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now']
Appending CPPFLAGS from environment: '-D_FORTIFY_SOURCE=2'
C compiler for the host machine: ccache cc (gcc 9.2.0 "cc (GCC) 9.2.0")
C linker for the host machine: GNU ld.bfd 2.33.1
Vala compiler for the host machine: valac (valac 0.46.3)
Host machine cpu family: x86_64
Host machine cpu: x86_64
Found pkg-config: /usr/bin/pkg-config (1.6.3)
Run-time dependency switchboard-2.0 found: YES 2.3.6
Run-time dependency gconf-2.0 found: YES 3.2.6
Run-time dependency gee-0.8 found: YES 0.20.2
Run-time dependency glib-2.0 found: YES 2.62.2
Run-time dependency granite found: YES 5.2.5
Run-time dependency gtk+-3.0 found: YES 3.24.12
Run-time dependency polkit-gobject-1 found: YES 0.116
Library posix found: YES
Program meson/post_install.py found: YES (/usr/bin/python /build/switchboard-plug-elementary-tweaks-git/src/switchboard-plug-elementary-tweaks/meson/post_install.py)
Build targets in project: 4
Found ninja-1.9.0 at /usr/bin/ninja
ninja: Entering directory `build'
[1/28] Compiling Vala source ../src/Tweaks.vala ../.../Widgets/SettingBox.vala ../src/Widgets/Label.vala.
FAILED: elementary-tweaks@sha/src/Tweaks.c elementary-tweaks@sha/src/Util.c elementary-tweaks@sha/src/Settings/AudienceSettings.c elementary-tweaks@sha/src/Settings/IndicatorSoundSettings.c elementary-tweaks@sha/src/Settings/FilesSettings.c elementary-tweaks@sha/src/Settings/GalaSettings.c elementary-tweaks@sha/src/Settings/GtkSettings.c elementary-tweaks@sha/src/Settings/PlankSettings.c elementary-tweaks@sha/src/Settings/KeyboardSettings.c elementary-tweaks@sha/src/Settings/SlingshotSettings.c elementary-tweaks@sha/src/Settings/GnomeSettings.c elementary-tweaks@sha/src/Settings/TerminalSettings.c elementary-tweaks@sha/src/Settings/XSettings.c elementary-tweaks@sha/src/Panes/AudiencePane.c elementary-tweaks@sha/src/Panes/AnimationsPane.c elementary-tweaks@sha/src/Panes/AppearancePane.c elementary-tweaks@sha/src/Panes/FontsPane.c elementary-tweaks@sha/src/Panes/MiscPane.c elementary-tweaks@sha/src/Panes/FilesPane.c elementary-tweaks@sha/src/Panes/TerminalPane.c elementary-tweaks@sha/src/Panes/PlankPane.c elementary-tweaks@sha/src/Panes/SlingshotPane.c elementary-tweaks@sha/src/Objects/ShortcutObject.c elementary-tweaks@sha/src/Widgets/Categories.c elementary-tweaks@sha/src/Widgets/SettingBox.c elementary-tweaks@sha/src/Widgets/Label.c elementary-tweaks.h elementary-tweaks.vapi 
valac -C --pkg posix --pkg switchboard-2.0 --pkg posix --pkg polkit-gobject-1 --pkg gtk+-3.0 --pkg granite --pkg glib-2.0 --pkg gee-0.8 --pkg gconf-2.0 --color=always --directory elementary-tweaks@sha --basedir ../ --library elementary-tweaks --header elementary-tweaks.h --vapi ../elementary-tweaks.vapi ../src/Tweaks.vala ../src/Util.vala ../src/Settings/AudienceSettings.vala ../src/Settings/IndicatorSoundSettings.vala ../src/Settings/FilesSettings.vala ../src/Settings/GalaSettings.vala ../src/Settings/GtkSettings.vala ../src/Settings/PlankSettings.vala ../src/Settings/KeyboardSettings.vala ../src/Settings/SlingshotSettings.vala ../src/Settings/GnomeSettings.vala ../src/Settings/TerminalSettings.vala ../src/Settings/XSettings.vala ../src/Panes/AudiencePane.vala ../src/Panes/AnimationsPane.vala ../src/Panes/AppearancePane.vala ../src/Panes/FontsPane.vala ../src/Panes/MiscPane.vala ../src/Panes/FilesPane.vala ../src/Panes/TerminalPane.vala ../src/Panes/PlankPane.vala ../src/Panes/SlingshotPane.vala ../src/Objects/ShortcutObject.vala ../src/Widgets/Categories.vala ../src/Widgets/SettingBox.vala ../src/Widgets/Label.vala
switchboard-2.0.vapi:14.3-14.13: warning: Creation method of abstract class cannot be public.
		public Plug ();
		^^^^^^^^^^^
../src/Widgets/Categories.vala:100.13-100.23: error: Creation method of abstract class cannot be public.
            public Pane (string label_string, string icon_name) {
            ^^^^^^^^^^^
Compilation failed: 1 error(s), 1 warning(s)
ninja: build stopped: subcommand failed.
```

--- a/src/Widgets/Categories.vala
+++ b/src/Widgets/Categories.vala
@@ -97,7 +97,7 @@ namespace ElementaryTweaks {
             public Widgets.SettingsBox settings_box { public get; private set; }
             protected Gtk.LinkButton reset;
 
-            public Pane (string label_string, string icon_name) {
+            protected Pane (string label_string, string icon_name) {
                 label.label = label_string;
                 image.icon_name = icon_name;
             }

